docs/versioned_docs/version-v1.3.0/02-Features/02-message-handling/01-service-bus.md

```diff
@@ -0,0 +1,599 @@
+﻿---
+title: "Azure Service Bus message handling"
+layout: default
+---
+
+# Azure Service Bus message handling
+The `Arcus.Messaging.Pumps.ServiceBus` library provides ways to perform all the plumbing that is required for processing messages on queues and topic subscriptions. 
+
+As a user, the only thing you have to do is **focus on processing messages, not how to get them**. Following terms are used:
+- **Message handler**: implementation that processes the received message from an Azure Service Bus queue or topic subscription. Message handlers are created by implementing the `IAzureServiceBusMessageHandler<TMessage>`. This message handler will be called upon when a message is available in the Azure Service Bus queue or on the topic subscription. [this section](#message-handler-example) for a message handler example setup
+- **Message router**: implementation that delegates the received Azure Service Bus message to the correct message handler. For alternative message routing, see [this section](#alternative-service-bus-message-routing) for more information.
+- **Message pump**: implementation that interacts and receives the Azure Service Bus message. The pump can be configured for different scenarios, see [this section](#pump-configuration) for more information.
+
+![Message handling schema](/media/worker-message-handling.png)
+
+## Installation
+This features requires to install our NuGet package:
+
+```shell
+PM > Install-Package Arcus.Messaging.Pumps.ServiceBus
+```
+
+> ⚠ The new Azure SDK doesn't yet support Azure Service Bus plugins. See this [migration guide](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md#known-gaps-from-previous-library) for more info on this topic.
```

Comment:
It could be used in our message pump by overriding some method, but since the new Azure SDK, the plugins are not supported yet but we can't remove the method as it would break the signature. Therefor we added this.